### PR TITLE
Saleor 1515/customise order confirmation messages

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds custom thank you subtitle depending on order status

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://feature-order-confirmation.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1515 Display different message on checkout "Thank you" page depending on confirmation](https://app.clickup.com/t/2549495/SALEOR-1515)

**Review by krzysztofwolski — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@editorjs/list": "^1.6.0",
     "@editorjs/quote": "^2.4.0",
     "@lhci/cli": "^0.4.1",
-    "@saleor/sdk": "^0.3.0-2",
+    "@saleor/sdk": "git://github.com/mirumee/saleor-sdk.git#update-order-status-type",
```
Comment (orzechdev) [inline package.json:17]: We should have only released sdk version on master

**Review by orzechdev — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (krzysztofwolski) [reply package.json:17]: @orzechdev it was feature branch, we know it :)

Comment (krzysztofwolski) [inline package.json:17]:
```suggestion
    "@saleor/sdk": "^0.3.0-3",
```

**Pull request merged**